### ISSUE TITLE
Brave in sandboxie plus occasionally takes system down

### ISSUE THREAD

**Issue by U17R** [sandboxie-plus/Sandboxie#589]
Brave version: Version 1.20.103 Chromium: 88.0.4324.152 (Official Build) (64-bit)
Sandboxie 0.6.7

rule setting: restrictive (brave.exe only)
tabs open ~10

graphic card: RTX 2080. 3 attached monitors

occasionally i get a black flicker and an error box that the resolution is not supported (not sure if the box comes from sandboxie or brave.  if i dont reboot, the system will most likely hang soon after. this happens even if brave is minimized in the background and nothing is going on.
it happens only if brave runs in sandboxie. also it never happened with the old sandboxie (sophos branch). (using sandboxie for years)

not sure how to provide more debug information. let me know how i can help.
as far as i can tell this issue does not occur if i run brave without a sandbox.

**Comment by DavidXanatos** [comment]
Oh that does not sound good, could you please test old versions until you find one that still works,
its extremely helpfully to know which version introduced the issue.

Also do you have any windows kernel crash dumps? that would help as well.

**Comment by U17R** [comment]
unfortunately i wasnt able to get a crash dump yet asit crashes too quick after.
i used Sandboxie-Classic-x64-v5.46.4.exe without issue
issues started when moving to Sandboxie-Plus-x64-v0.6.5.exe
as i write this there was also an issue when upgrading to sandboxie plus:
it could not remove sbiedll.dll 
i tried to unlock it but it had no lock handler on it. thus i had to mark it for deletilon on a system restart and could install sandboxie plus only successfully after. not sure if thats related but i thought it may be worth mentioning.

**Comment by DavidXanatos** [comment]
is the issue still present?

**Comment by U17R** [comment]
no, it disappeared after one of the updates i guess. (no changes
      on the system side here)
    
    On 07/06/2021 20:08, DavidXanatos
      wrote:
    
    
      
      is the issue still present?
      —
        You are receiving this because you authored the thread.
        Reply to this email directly, view it on GitHub, or unsubscribe.
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/sandboxie-plus/Sandboxie/issues/589#issuecomment-856151208",
"url": "https://github.com/sandboxie-plus/Sandboxie/issues/589#issuecomment-856151208",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by DavidXanatos** [comment]
ok great than i can close it
